nginx/sites-enabled/hashbang-http

```diff
@@ -0,0 +1,18 @@
+# Hashbang server configuration
+# ref: https://gist.github.com/RyanSquared/e59c7e274abef06e63b0e47e46997b00
+#
+server {
+	listen 80;
+	listen [::]:80;
+
+	server_name "~^(?<user>[a-zA-Z0-9]+)\.(?<server>\w{2}\d{1})\.hashbang\.sh$";
+
+	root /home/$user/Public;
```

Comment:
Well, we can redirect `server.hashbang.sh/~user` to `user.hashbang.sh`.

If you mean no automatic reading from `~/Public`, yes (for the reasons discussed on IRC).
However, we can add to `dotfiles` a default systemd unit that listens on `/var/run/${uid}/http.socket` and exposes `~/Public` (ideally, with socket activation).